### DISCUSSION TITLE
Auto focus on the search box when the extension icon is clicked

### DISCUSSION THREAD

**Discussion by deepakmahakale** [Authenticator-Extension/Authenticator#1131] [Ideas]
Most of the time when I open the website in another window and want to get the code I found myself typing the site name to realise I wasn't typing in the search box.

Can we auto-focus on the search box so that when it's opened I can just type the site name

**Comment by TriMoon** [comment]
It's totally UNCLEAR, to me, what you want (WHY) with the way you are describing your issue...
I personally would NOT want to be auto focused on the search bar when i click the extension's icon to select the 2FA code...
